plotting-visualization/rtdose_visual.py

```diff
@@ -0,0 +1,265 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Mar 31 10:45:06 2021
+
+@author: Francisco Piqueras Guardiola
+@author: Fernando Hueso González
+
+Note: This code is designed to be run in spyder environment
+"""
+import pydicom
+import matplotlib.pyplot as plt
+import os
+import numpy as np
+import gzip
+import sys
+
+#!/usr/bin/python
+
+
+# Here we define the functions that we are gona use in our program
```

Comment:
@Kikooku as Darcy suggested, please change "gona" with "going to". Thanks!!

Reply:
Done!